chapters/ch07.asciidoc

```diff
@@ -1419,9 +1419,9 @@ const things = {
   length: 2
 }
 Array.from(things)
-// <- [<div>, <div>, <div>, ...]
+// <- [Object, Object]
```

Comment:
What do you mean? `Array.from(things, thing => thing.name)` would give `['Apple', 'Onion']`, but would be a bit redundant. But I don't think that's what you mean. Can you give me an example?